coordinator/persistence-api/src/main/java/org/apache/cassandra/stargate/exceptions/RequestFailureReason.java

```diff
@@ -46,7 +50,7 @@ public enum RequestFailureReason {
    * between the operation and either creation or drop of the table (or, possibly, of the keyspace
    * containing that table, see comment on {@link #UNKNOWN_KEYSPACE}).
    */
-  UNKNOWN_TABLE,
+  UNKNOWN_TABLE(501),
```

Comment:
`codeMap` has now size 500, should we improve?

Reply:
Probably fine at this point, changing implementation would add risk. 500 x 4 (or 8) bytes is still pretty small chunk of memory?

Reply:
Sounds good: I like the idea of codebases being in closer sync.